### PR TITLE
Add Explore Card Item, Bouncing Atoms, and About Page

### PR DESCRIPTION
**Pull request by Yukino2002**

### CODE REVIEW

File: new-website/deepchem/components/Home/BouncingAtoms/BouncingAtoms.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+ * function to form bond between two atoms
+ * @function
+ *
+ * @param {Atom} a - first atom
+ * @param {Atom} b - second atom
+ * @param {number} separation - distance between the two atoms
+ * @param {object} p5 - reference to p5 object to access p5 functions
+ */
+function formBond(a, b, separation, p5) {
+  p5.stroke(p5.color(192, 192, 192));
```
Comment (rbharath) [inline new-website/deepchem/components/Home/BouncingAtoms/BouncingAtoms.js:118]: Can you add a comment about what color this is?

File: new-website/deepchem/components/Home/BouncingAtoms/BouncingAtoms.styles.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+html, body {
+  margin: 0;
+  padding: 0;
+}
+canvas {
+  display: block;
+  background: rgb(0,78,152);
+  background: linear-gradient(35deg, rgba(0,78,152,1) 0%, rgba(58,110,165,1) 34%, rgba(255,103,0,1) 100%);
```
Comment (rbharath) [inline new-website/deepchem/components/Home/BouncingAtoms/BouncingAtoms.styles.css:15]: Can we add comments saying what these colors are?

File: new-website/deepchem/pages/about.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+import React from "react";
+import DeveloperDeskImage from "../public/images/about/developer.png";
+
+import faqs from "../data/about/faqs.json";
+import CustomCarousel from "./../components/CustomCarousel/CustomCarousel";
+import CarouselItem from "./../components/CustomCarousel/CarouselItem";
+
+import fromDevelopersDeskTest from "../data/about/from-developers-desk.json";
+
+const loadContributersCarouselImageData = () => {
```
Comment (rbharath) [inline new-website/deepchem/pages/about.js]: Add docstring

**Review by rbharath — COMMENTED** [review]
Code is good but should be broken up into a few PR:

- Bouncing atoms
- About page updates
- Explore Card / Logo

File: new-website/deepchem/components/Home/BouncingAtoms/BouncingAtoms.js (hunk 1 above)
Comment (dibyam1101) [reply new-website/deepchem/components/Home/BouncingAtoms/BouncingAtoms.js:118]: Added

File: new-website/deepchem/components/Home/BouncingAtoms/BouncingAtoms.styles.css (hunk 2 above)
Comment (dibyam1101) [reply new-website/deepchem/components/Home/BouncingAtoms/BouncingAtoms.styles.css:15]: Added

**Review by rbharath — APPROVED** [review]
LGTM

**Pull request merged**